bosh-setup/manifests/multiple-vm-cf.yml

```diff
@@ -320,6 +456,153 @@ properties:
     port: 5524
     roles:
     - {name: ccadmin, password: c1oudc0w, tag: admin}
+  consul:
+    agent:
+      log_level: null
+      servers:
+        lan:
+        - REPLACE_WITH_CONSUL_IP
+    agent_cert: |+
+      -----BEGIN CERTIFICATE-----
+      MIIEITCCAgugAwIBAgIQTDMCJHBiQrY8hlM78doHTzALBgkqhkiG9w0BAQswEzER
```

Comment:
Yeah let's leave comments or hints as to the nature of these and where they are used to remove any confusion for users/security team not familiar with Cloudfoundry/Bosh.